### PR TITLE
Add support for Southpaw Extended 65%

### PR DESCRIPTION
**Pull request by horza21**
## Description

Added support for Southpaw Extended 65%

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/switchplate/southpaw_65/rules.mk [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+LINK_TIME_OPTIMIZATION_ENABLE = yes
+
+# custom matrix setup
+CUSTOM_MATRIX = lite
+
+VPATH += drivers/gpio
+SRC += pca9555.c matrix.c
+QUANTUM_LIB_SRC += i2c_master.c
+
+LAYOUTS = default default2 default_iso default_iso2
```
Comment (zvecr) [inline keyboards/switchplate/southpaw_65/rules.mk]: This is for <https://docs.qmk.fm/#/feature_layouts>, not listing every layout macro you have.
```suggestion
```

File: keyboards/switchplate/southpaw_65/southpaw_65.c [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+void led_set_kb(uint8_t usb_led) {
+  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+    writePinLow(B6);
+  } else {
+    writePinHigh(B6);
+  }
+
+  led_set_user(usb_led);
+}
```
Comment (zvecr) [inline keyboards/switchplate/southpaw_65/southpaw_65.c]:
```suggestion
bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if(res) {
        writePin(B6, !led_state.caps_lock);
    }
    return res;
}
```

File: keyboards/switchplate/southpaw_65/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- iso and ansi support
+- Uses Kimera core
+
+Keyboard Maintainer: QMK Community
+Hardware Supported: XD85
+Hardware Availability: [Geekhack GB Thread](https://geekhack.org/index.php?topic=92344.0)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make southpaw_65:default
```
Comment (zvecr) [inline keyboards/switchplate/southpaw_65/readme.md]:
```suggestion
    make switchplate/southpaw_65:default
```

File: keyboards/switchplate/southpaw_65/readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+Install examples:
+
+    make southpaw_65:default:dfu
```
Comment (zvecr) [inline keyboards/switchplate/southpaw_65/readme.md]:
```suggestion
    make switchplate/southpaw_65:default:flash
```

File: keyboards/switchplate/southpaw_65/readme.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+Left Hand Number Pad Keyboard with 85 Keys & RGB LED Underglow
+- Designed by Reconsiderit
+- Up to 90 keys
+- iso and ansi support
+- Uses Kimera core
+
+Keyboard Maintainer: QMK Community
+Hardware Supported: XD85
+Hardware Availability: [Geekhack GB Thread](https://geekhack.org/index.php?topic=92344.0)
```
Comment (zvecr) [inline keyboards/switchplate/southpaw_65/readme.md]:
```suggestion
* Keyboard Maintainer: QMK Community
* Hardware Supported: Southpaw Extended 65%
* Hardware Availability: [Geekhack GB Thread](https://geekhack.org/index.php?topic=92344.0)
```

**Comment by horza21** [comment]
> Also as a general comment, i'm not a fan of the ansi2/iso2 LAYOUT macro names. Though i dont have a better suggestion, we try to align them with functionality (add stuff like` _split_bs` for split backspace)

I was wondering what to name those alternate layouts too. I'm open to suggestions if there's a convention for the 2x1.5u vs the 3x1u layouts.

**Comment by fauxpark** [comment]
We have a PCA9555 driver under drivers/gpio - would it be possible to use that to cut down on the code duplication here?

File: keyboards/switchplate/southpaw_65/info.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
+    "keyboard_name": "Southpaw Extended 65%",
+    "url": "https://geekhack.org/index.php?topic=92344.0",
+    "maintainer": "qmk",
+    "width": 20,
+    "height": 5,
+    "layouts": {
+    	  "LAYOUT_ansi": {
```
Comment (noroadsleft) [inline keyboards/switchplate/southpaw_65/info.json]:
```suggestion
    	  "LAYOUT_all": {
```

File: keyboards/switchplate/southpaw_65/info.json [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
...y":2}, {"label":"\"", "x":15.75, "y":2}, {"label":"~", "x":16.75, "y":2}, {"label":"Page Up", "x":19, "y":2}, {"label":"1", "x":0, "y":3}, {"label":"2", "x":1, "y":3}, {"label":"3", "x":2, "y":3}, {"label":"Num Enter", "x":3, "y":3, "h":2}, {"label":"LShift", "x":4, "y":3, "w":1.25}, {"label":"|", "x":5.25, "y":3}, {"label":"Z", "x":6.25, "y":3}, {"label":"X", "x":7.25, "y":3}, {"label":"C", "x":8.25, "y":3}, {"label":"V", "x":9.25, "y":3}, {"label":"B", "x":10.25, "y":3}, {"label":"N", "x":11.25, "y":3}, {"label":"M", "x":12.25, "y":3}, {"label":"<", "x":13.25, "y":3}, {"label":">", "x":14.25, "y":3}, {"label":"?", "x":15.25, "y":3}, {"label":"RShift", "x":16.25, "y":3, "w":1.75}, {"label":"Up", "x":18, "y":3}, {"label":"Page Down", "x":19, "y":3}, {"label":"0", "x":0, "y":4, "w":2}, {"label":".", "x":2, "y":4}, {"label":"LCtrl", "x":4, "y":4, "w":1.25}, {"label":"Win", "x":5.25, "y":4, "w":1.25}, {"label":"LAlt", "x":6.5, "y":4, "w":1.25}, {"label":"Space", "x":7.75, "y":4, "w":6.25}, {"label":"RAlt", "x":14, "y":4}, {"label":"RCtrl", "x":15, "y":4}, {"label":"Fn0", "x":16, "y":4}, {"label":"Left", "x":17, "y":4}, {"label":"Down", "x":18, "y":4}, {"label":"Right", "x":19, "y":4}]
```
Comment (noroadsleft) [inline keyboards/switchplate/southpaw_65/info.json]: ISO Enter's object is out of sequence here.

```suggestion
            "layout": [{"label":"Num Lock", "x":0, "y":0}, {"label":"/", "x":1, "y":0}, {"label":"*", "x":2, "y":0}, {"label":"-", "x":3, "y":0}, {"label":"Esc", "x":4, "y":0}, {"label":"!", "x":5, "y":0}, {"label":"@", "x":6, "y":0}, {"label":"#", "x":7, "y":0}, {"label":"$", "x":8, "y":0}, {"label":"%", "x":9, "y":0}, {"label":"^", "x":10, "y":0}, {"label":"&", "x":11, "y":0}, {"label":"*", "x":12, "y":0}, {"label":"(", "x":13, "y":0}, {"label":")", "x":14, "y":0}, {"label":"_", "x":15, "y":0}, {"label":"+", "x":16, "y":0}, {"label":"Backspace", "x":17, "y":0, "w":2}, {"label":"Home", "x":19, "y":0}, {"label":"7", "x":0, "y":1}, {"label":"8", "x":1, "y":1}, {"label":"9", "x":2, "y":1}, {"label":"+", "x":3, "y":1, "h":2}, {"label":"Tab", "x":4, "y":1, "w":1.5}, {"label":"Q", "x":5.5, "y":1}, {"label":"W", "x":6.5, "y":1}, {"label":"E", "x":7.5, "y":1}, {"label":"R", "x":8.5, "y":1}, {"label":"T", "x":9.5, "y":1}, {"label":"Y", "x":10.5, "y":1}, {"label":"U", "x":11.5, "y":1}, {"label":"I", "x":12.5, "y":1}, {"label":"O", "x":13.5, "y":1}, {"label":"P", "x":14.5, "y":1}, {"label":"{", "x":15.5, "y":1}, {"label":"}", "x":16.5, "y":1}, {"label":"End", "x":19, "y":1}, {"label":"4", "x":0, "y":2}, {"label":"5", "x":1, "y":2}, {"label":"6", "x":2, "y":2}, {"label":"Caps Lock", "x":4, "y":2, "w":1.75}, {"label":"A", "x":5.75, "y":2}, {"label":"S", "x":6.75, "y":2}, {"label":"D", "x":7.75, "y":2}, {"label":"F", "x":8.75, "y":2}, {"label":"G", "x":9.75, "y":2}, {"label":"H", "x":10.75, "y":2}, {"label":"J", "x":11.75, "y":2}, {"label":"K", "x":12.75, "y":2}, {"label":"L", "x":13.75, "y":2}, {"label":":", "x":14.75, "y":2}, {"label":"\"", "x":15.75, "y":2}, {"label":"~", "x":16.75, "y":2}, {"label":"Enter", "x":17.75, "y":1, "w":1.25, "h":2}, {"label":"Page Up", "x":19, "y":2}, {"label":"1", "x":0, "y":3}, {"label":"2", "x":1, "y":3}, {"label":"3", "x":2, "y":3}, {"label":"Num Enter", "x":3, "y":3, "h":2}, {"label":"LShift", "x":4, "y":3, "w":1.25}, {"label":"|", "x":5.25, "y":3}, {"label":"Z", "x":6.25, "y":3}, {"label":"X", "x":7.25, "y":3}, {"label":"C", "x":8.25, "y":3}, {"label":"V", "x":9.25, "y":3}, {"label":"B", "x":10.25, "y":3}, {"label":"N", "x":11.25, "y":3}, {"label":"M", "x":12.25, "y":3}, {"label":"<", "x":13.25, "y":3}, {"label":">", "x":14.25, "y":3}, {"label":"?", "x":15.25, "y":3}, {"label":"RShift", "x":16.25, "y":3, "w":1.75}, {"label":"Up", "x":18, "y":3}, {"label":"Page Down", "x":19, "y":3}, {"label":"0", "x":0, "y":4, "w":2}, {"label":".", "x":2, "y":4}, {"label":"LCtrl", "x":4, "y":4, "w":1.25}, {"label":"Win", "x":5.25, "y":4, "w":1.25}, {"label":"LAlt", "x":6.5, "y":4, "w":1.25}, {"label":"Space", "x":7.75, "y":4, "w":6.25}, {"label":"RAlt", "x":14, "y":4}, {"label":"RCtrl", "x":15, "y":4}, {"label":"Fn0", "x":16, "y":4}, {"label":"Left", "x":17, "y":4}, {"label":"Down", "x":18, "y":4}, {"label":"Right", "x":19, "y":4}]
```

File: keyboards/switchplate/southpaw_65/readme.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+# Southpaw Extended 65%
+
+![Southpaw Extended 65%](https://i.imgur.com/9HSWmml.jpg)
```
Comment (noroadsleft) [inline keyboards/switchplate/southpaw_65/readme.md]: Using a dimensionally-smaller image.

```suggestion
![Southpaw Extended 65%](https://i.imgur.com/9HSWmmll.jpg)
```

File: keyboards/switchplate/southpaw_65/rules.mk [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```
Comment (noroadsleft) [inline keyboards/switchplate/southpaw_65/rules.mk]: Same rule, but less unwieldy.

```suggestion
LTO_ENABLE = yes
```

**Comment by horza21** [comment]
> Also, you need to have a `keyboards/switchplate/southpaw_65/keymaps/default/keymap.c` that uses the `LAYOUT_all` macro.

Ok. I'll add that in when I'm able to.

> We have a PCA9555 driver under drivers/gpio - would it be possible to use that to cut down on the code duplication here?

I'm not trying to reimplement the PCA9555 driver. I'm just linking to the driver from matrix.c.
Unless there's something I'm not seeing here.

**Comment by fauxpark** [comment]
> I'm not trying to reimplement the PCA9555 driver. I'm just linking to the driver from matrix.c.
> Unless there's something I'm not seeing here.

...I could have sworn that wasn't the case. My bad, looks good 👍

File: keyboards/switchplate/southpaw_65/keymaps/default/keymap.c [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+/* ├─────────┼─────────┼─────────┼─────────┼─────────────────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───────────┼─────────┼─────────┤ */
+	  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,      KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,            KC_NO,    KC_NO,
+/* ├─────────┼─────────┼─────────┼─────────┼───────────┼─────────┴─┬───────┴───┬─────┴─────────┴─────────┴─────────┴─────────┴─────────┴───────┬─┴───────┬─┴───────┬─┴───────┬─────────┼─────────┼─────────┤ */
+	  KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,      KC_NO,      KC_NO,                             KC_NO,                                   KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO,    KC_NO
+/* └─────────┴─────────┴─────────┴─────────┴───────────┴───────────┴───────────┴───────────────────────────────────────────────────────────────┴─────────┴─────────┴─────────┴─────────┴─────────┴─────────┘ */
```
Comment (noroadsleft) [inline keyboards/switchplate/southpaw_65/keymaps/default/keymap.c]: Users are likely to copy this file to make their keymaps, so it should be usable.

```suggestion
/* ┌─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┐ */
    KC_NLCK,  KC_PSLS,  KC_PAST,  KC_PMNS,  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_BSPC,  KC_HOME,
/* ├─────────┼─────────┼─────────┼─────────┼─────────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴─────────┼─────────┤ */
    KC_P7,    KC_P8,    KC_P9,    KC_PPLS,  KC_TAB,        KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,       KC_END,
/* ├─────────┼─────────┼─────────┼─────────┼──────────────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬───────────┼─────────┤ */
    KC_P4,    KC_P5,    KC_P6,    KC_PPLS,  KC_CAPS,          KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_NUHS,  KC_ENT,     KC_PGUP,
/* ├─────────┼─────────┼─────────┼─────────┼───────────┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴─────────┴─┬─────────┼─────────┤ */
    KC_P1,    KC_P2,    KC_P3,    KC_PENT,  KC_LSFT,    KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,          KC_UP,    KC_PGDN,
/* ├─────────┼─────────┼─────────┼─────────┼───────────┼─────────┴─┬───────┴───┬─────┴─────────┴─────────┴─────────┴─────────┴─────────┴───────┬─┴───────┬─┴───────┬─┴───────┬─────────┼─────────┼─────────┤ */
    KC_P0,    KC_P0,    KC_PDOT,  KC_PENT,  KC_LCTL,    KC_LGUI,    KC_LALT,                            KC_SPC,                                 KC_RALT,  KC_RCTL,  MO(1),    KC_LEFT,  KC_DOWN,  KC_RGHT
/* └─────────┴─────────┴─────────┴─────────┴───────────┴───────────┴───────────┴───────────────────────────────────────────────────────────────┴─────────┴─────────┴─────────┴─────────┴─────────┴─────────┘ */
    ),

    [1] = LAYOUT_all(
/* ┌─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┬─────────┐ */
    _______,  _______,  _______,  _______,  KC_GRV,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_DEL,   KC_DEL,   KC_INS,
/* ├─────────┼─────────┼─────────┼─────────┼─────────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴────┬────┴─────────┼─────────┤ */
    _______,  _______,  _______,  _______,  RESET,         BL_TOGG,  BL_INC,    BL_DEC,  BL_BRTG,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,       _______,
/* ├─────────┼─────────┼─────────┼─────────┼──────────────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬──────┴──┬───────────┼─────────┤ */
    _______,  _______,  _______,  _______,  _______,          RGB_TOG,  RGB_HUI,  RGB_SAI,  RGB_VAI,  RGB_SPI,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  EEP_RST,    _______,
/* ├─────────┼─────────┼─────────┼─────────┼───────────┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴───┬─────┴─────────┴─┬─────────┼─────────┤ */
    _______,  _______,  _______,  _______,  KC_LSFT,    _______,  RGB_MOD,  RGB_HUD,  RGB_SAD,  RGB_VAD,  RGB_SPD,  _______,  _______,  _______,  _______,  _______,  KC_RSFT,          _______,  _______,
/* ├─────────┼─────────┼─────────┼─────────┼───────────┼─────────┴─┬───────┴───┬─────┴─────────┴─────────┴─────────┴─────────┴─────────┴───────┬─┴───────┬─┴───────┬─┴───────┬─────────┼─────────┼─────────┤ */
    _______,  _______,  _______,  _______,  KC_LCTL,    _______,    KC_LALT,                            _______,                                KC_RALT,  KC_RCTL,  KC_TRNS,  _______,  _______,  _______
/* └─────────┴─────────┴─────────┴─────────┴───────────┴───────────┴───────────┴───────────────────────────────────────────────────────────────┴─────────┴─────────┴─────────┴─────────┴─────────┴─────────┘ */
```

File: keyboards/switchplate/southpaw_65/info.json [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+        },
```
Comment (noroadsleft) [inline keyboards/switchplate/southpaw_65/info.json]:
```suggestion
        "LAYOUT_all": {
            "key_count": 91,
            "layout": [{"label":"Num Lock", "x":0, "y":0}, {"label":"/", "x":1, "y":0}, {"label":"*", "x":2, "y":0}, {"label":"-", "x":3, "y":0}, {"label":"Esc", "x":4, "y":0}, {"label":"!", "x":5, "y":0}, {"label":"@", "x":6, "y":0}, {"label":"#", "x":7, "y":0}, {"label":"$", "x":8, "y":0}, {"label":"%", "x":9, "y":0}, {"label":"^", "x":10, "y":0}, {"label":"&", "x":11, "y":0}, {"label":"*", "x":12, "y":0}, {"label":"(", "x":13, "y":0}, {"label":")", "x":14, "y":0}, {"label":"_", "x":15, "y":0}, {"label":"+", "x":16, "y":0}, {"label":"Backspace", "x":17, "y":0}, {"label":"Backspace", "x":17, "y":0}, {"label":"Home", "x":19, "y":0}, {"label":"7", "x":0, "y":1}, {"label":"8", "x":1, "y":1}, {"label":"9", "x":2, "y":1}, {"label":"+", "x":3, "y":1, "h":2}, {"label":"Tab", "x":4, "y":1, "w":1.5}, {"label":"Q", "x":5.5, "y":1}, {"label":"W", "x":6.5, "y":1}, {"label":"E", "x":7.5, "y":1}, {"label":"R", "x":8.5, "y":1}, {"label":"T", "x":9.5, "y":1}, {"label":"Y", "x":10.5, "y":1}, {"label":"U", "x":11.5, "y":1}, {"label":"I", "x":12.5, "y":1}, {"label":"O", "x":13.5, "y":1}, {"label":"P", "x":14.5, "y":1}, {"label":"{", "x":15.5, "y":1}, {"label":"}", "x":16.5, "y":1}, {"label":"|", "x":17.5, "y":1, "w":1.5}, {"label":"End", "x":19, "y":1}, {"label":"4", "x":0, "y":2}, {"label":"5", "x":1, "y":2}, {"label":"6", "x":2, "y":2}, {"label":"Caps Lock", "x":4, "y":2, "w":1.75}, {"label":"A", "x":5.75, "y":2}, {"label":"S", "x":6.75, "y":2}, {"label":"D", "x":7.75, "y":2}, {"label":"F", "x":8.75, "y":2}, {"label":"G", "x":9.75, "y":2}, {"label":"H", "x":10.75, "y":2}, {"label":"J", "x":11.75, "y":2}, {"label":"K", "x":12.75, "y":2}, {"label":"L", "x":13.75, "y":2}, {"label":":", "x":14.75, "y":2}, {"label":"\"", "x":15.75, "y":2}, {"label":"~", "x":16.75, "y":2}, {"label":"Enter", "x":17.75, "y":2, "w":1.25}, {"label":"Page Up", "x":19, "y":2}, {"label":"1", "x":0, "y":3}, {"label":"2", "x":1, "y":3}, {"label":"3", "x":2, "y":3}, {"label":"Num Enter", "x":3, "y":3, "h":2}, {"label":"LShift", "x":4, "y":3, "w":1.25}, {"label":"|", "x":5.25, "y":3}, {"label":"Z", "x":6.25, "y":3}, {"label":"X", "x":7.25, "y":3}, {"label":"C", "x":8.25, "y":3}, {"label":"V", "x":9.25, "y":3}, {"label":"B", "x":10.25, "y":3}, {"label":"N", "x":11.25, "y":3}, {"label":"M", "x":12.25, "y":3}, {"label":"<", "x":13.25, "y":3}, {"label":">", "x":14.25, "y":3}, {"label":"?", "x":15.25, "y":3}, {"label":"RShift", "x":16.25, "y":3, "w":1.75}, {"label":"Up", "x":18, "y":3}, {"label":"Page Down", "x":19, "y":3}, {"label":"0", "x":0, "y":4}, {"label":"0", "x":1, "y":4}, {"label":".", "x":2, "y":4}, {"label":"LCtrl", "x":4, "y":4, "w":1.25}, {"label":"Win", "x":5.25, "y":4, "w":1.25}, {"label":"LAlt", "x":6.5, "y":4, "w":1.25}, {"label":"Space", "x":7.75, "y":4, "w":6.25}, {"label":"RAlt", "x":14, "y":4}, {"label":"RCtrl", "x":15, "y":4}, {"label":"Fn0", "x":16, "y":4}, {"label":"Left", "x":17, "y":4}, {"label":"Down", "x":18, "y":4}, {"label":"Right", "x":19, "y":4}]
        },
```

**Comment by noroadsleft** [comment]
Thanks!

**Comment by horza21** [comment]
> Thanks!

Thank you too!

**Pull request merged**